MasterPassword/C/mpw-cli.c

```diff
@@ -210,21 +210,26 @@ int main(int argc, char *const argv[]) {
         masterPassword = getpass( "Your master password: " );
 
     // Summarize operation.
-    fprintf( stderr, "%s's password for %s:\n[ %s ]: ", fullName, siteName, mpw_identicon( fullName, masterPassword ) );
+    char const * identicon = mpw_identicon( fullName, masterPassword );
+    fprintf( stderr, "%s's password for %s:\n[ %s ]: ", fullName, siteName, identicon );
+	mpw_freeString( identicon );
 
     // Output the password.
     const uint8_t *masterKey = mpw_masterKeyForUser(
             fullName, masterPassword, algorithmVersion );
     mpw_freeString( masterPassword );
+    mpw_freeString( fullName );
```

Comment:
Yes. `getenv` always returns `NULL` and I do not  use the `-u` option, then I always have memory leak.
Sorry for delay in answering, I was traveling.